Mixpanel/MPNotificationViewController.m

```diff
@@ -269,6 +285,11 @@ - (void)viewDidLoad
     [self.view addGestureRecognizer:pan];
 }
 
+- (void)initializeMiniNotification {
+    self.backgroundColor = [UIColor colorWithWhite:0 alpha:0.901f];
```

Comment:
Trying to be precise with how Mason did it lol. He had a 0.55 overlay and over it a 0.78 overlay